src/third-party/pinyin/pinyin_data.hpp

```diff
@@ -18,6 +19,7 @@
 #include <string>
 
 static const std::map<std::u32string, std::u32string> pinyin_data = {
+    {U"", U"㓘㟬㣇㤺㬈㱎㼔㼛䄯䇫䈧䣺䦕䩶冞劷叽埒宂宜岍嵮廾形惇憤戛揅斲曼枅椔檨殦涅潛瀛灊灰災牐犀珊甮竮籑緇罺聠舁茣荆蚈蚩蛌蛢裗豕賾跚鄑鄖鄛酁鐕韌頋飢鬒鼖𠂇𠒢𠓌𠢢𠣉𡙛𡥋𡬘𡶭𡽜𢍫𢔴𢛔𢻂𣝜𣽨𤕚𤜵𤳘𥌱𥖗𥟿𥩒𥪧𥳐𦈨𦣹𦥱𦫎𦾱𧈚𧗆𧙧𧢛𧲨𧷎𨊻𨋐𨗥𩄮𩲡𪆷"},
```

Comment:
Something probably went wrong here.

Reply:
Oh, I just forgot to remove this line, it is caused by those data entries that have two index number but one pronounciation, such as this one: 
![image](https://user-images.githubusercontent.com/43123946/220532694-e27e1a25-2616-4879-b295-634fd35cba0c.png)
My conversion code was just a makeshift one so it happens. Removed now